### ISSUE TITLE
Login not working

### ISSUE THREAD

**Issue by fnorddd** [EliasKotlyar/Xiaomi-Dafang-Hacks#1281]
Hi,
I dig up my old Xiaomi Xiaofang 1S T20 and followed the installation guide and configured the wifi.
The camera was not connected to the power since around 1-1.5 years.

Once the camera is connected to the wifi the following ports are open:

> 
> PORT     STATE SERVICE
> 
> 21/tcp   open  ftp
> 22/tcp   open  ssh
> 80/tcp   open  http
> 554/tcp  open  rtsp
> 2323/tcp open  3d-nfsd
> 
> MAC Address: CO:FF:EE:BA:BE (Xiaomi Electronics,co.)

When I connect via ssh, it gives me a "Access denied" with root/ismart12. 
Telnet (on 2332) gives me a promt "iSmartAlarm login:" but login is not working.

FTP login with root works without a password and shows me:
![grafik](https://user-images.githubusercontent.com/61168300/74691748-547b9500-51e4-11ea-956b-bf123f5c0e06.png)

In /var/log/messages I see:
_Feb 18 01:12:35 iSmartAlarm authpriv.warn dropbear[2362]: Bad password attempt for 'root' from 192.168.2.68:64646_

Any idea what I am doing wrong?

Thanks and regards

**Comment by jmtatsch** [comment]
Did you change your ssh password back then? Or just start from a fresh checkout of firmware_mod.

**Comment by fnorddd** [comment]
I only updated the wpa_supplicant.conf file, nothing else.
The rest is just out of the repo (firmware_mod).

**Comment by Dopeyr** [comment]
Hi, something sounds... odd... the open port list doesn't seem to match what the CFW provides. e.g. RTSP runs on 8554, to 554 and port 443 should be open. I don't know what service runs on 2323 - I can't find mention of it in the repo. "iSmartAlarm login" isn't familiar either. Maybe it's one of these cameras? https://www.ismartalarm.com/

**Comment by fnorddd** [comment]
It is for sure a Xiaomi one, but I see they were selling the same model (spot) under their own brand. 
Maybe I tried to flash this fw a while ago?!

What is the best way to flash this repos fw/bootloader if I dont get any response from the camera (as it is only LED blinking)?

**Comment by fnorddd** [comment]
![index](https://user-images.githubusercontent.com/61168300/74775003-112d2f00-5295-11ea-8f3c-9d497d970d61.jpg)

**Comment by fnorddd** [comment]
Okay, i followed down the rabbit hole and found out, that this camera is a licence product, bought by 3-4 different companies at least. 
While trying a few of my default passwords for non-internet exposed IoT devices, I got welcomed with `Welcome to XiaoFang Hacks :-)`.
It looks that I already flashed some alternative fw. Will find out, what needs to be done do flash a new one. Thanks for the support and sorry for confusing 👍
